controllers/infinispan_controller.go

```diff
@@ -1760,18 +1765,23 @@ func GossipRouterPodList(infinispan *infinispanv1.Infinispan, kube *kube.Kuberne
 	return podList, kube.ResourcesList(infinispan.Namespace, GossipRouterPodLabels(infinispan.Name), podList, ctx)
 }
 
-func buildStartupArgs(overlayConfigMapKey string, zeroCapacity string) []string {
-	var args []string
+func buildStartupArgs(overlayConfigMapKey, overlayLog4jConfigMapKey, zeroCapacity string) []string {
```

Comment:
As the file must always be named "log4j.xml` we can remove this parameter and just use the filename directly when building the args string.

Reply:
In this case, how could we then specify the correct path to the logging configuration file? It looks like the path in the `args` changes depending on the user providing a custom config file.
```
when user-provided --> user/log4j.xml
default --> /conf/operator/log4j.xml
```

Reply:
I was thinking that instead of passing `overlayLog4jConfigMapKey` we could pass the `overlayConfigMap` from `statefulSetForInfinispan` and check if the file exists in the CM. However, that doesn't take into account `updateStartupArgs` ... so I think it's better to stick with your original approach :+1:

Reply:
ok, updating the PR as discussed and will push it again when ready (for another round 🙂)